core/controllers/question_editor.py

```diff
@@ -114,10 +114,6 @@ def get(self, question_id):
         question = question_services.get_question_by_id(
             question_id, strict=False)
 
-        if question is None:
```

Comment:
Why was this and the check below removed? Granted there is a question_rights check in acl_decorators, but it might be good to have this as well, in case question_rights and questions get detached due to some reason.
What do you think?

Reply:
Yes I agree. I reverted the changes and created a mock function to handle this.